wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -0,0 +1,782 @@
+
+Es gibt keine visuellen Unterschiede zwischen externen und osu!-internen Links. Deshalb sollte der Name der Webseite im Linktext enthalten sein. Siehe das folgende Beispiel:
+
+```markdown
+*Für mehr Informationen über die Musiktheorie, siehe: [Musiktheorie](https://de.wikipedia.org/wiki/Musiktheorie)*
+```
+
+## Bilder
+
+Es gibt zwei Arten von Bildlinks: Inline und als Verweis. Beispiele:
+
+**Inline-Stil:**
+
+```markdown
+![Goldkrone](/wiki/shared/crown-gold.png "Erster Platz")
+```
+
+**Verweis-Stil:**
+
+```markdown
+![Goldkrone][GCrown]
+
+[GCrown]: /wiki/shared/crown-gold.png "Erster Platz"
+```
+
+Bilder sollten den Inline-Stil verwenden. Definitionen für Verweise müssen am Ende des Artikels platziert werden.
+
+### Alternative und Titeltexte
+
+Der Text in den ersten eckigen Klammern (*alternativer Text*) sollte das Bild wörtlich beschreiben. Dies wird von Bildschirmlesegeräten benutzt und wird angezeigt, falls das Bild nicht geladen werden kann. Es kann weggelassen werden, wenn es identisch mit dem Titeltext ist oder wenn das Bild nur aus dekorativen Zwecken enthalten ist.
+
+Der Text in Anführungszeichen (*Titeltext*) sollte zusätzlichen Kontext zum Bild geben oder seine Bedeutung wiedergeben. Es wird angezeigt als ein Tooltip, wenn man mit der Maus über das Bild fährt. Es kann auch als eine Bildunterschrift gesehen werden. Markdown-Formatierungen werden hier nicht unterstützt.
+
+### Anzeige
+
+Wenn ein Bild der einzige Inhalt eines Absatzes ist, dann wird dieses als zentrierter Block angezeigt. Ansonsten liegt es dynamisch im umgebenden Inline-Text.
+
+Blockbilder mit einem Titeltext zeigen den Titeltext als eine Bildunterschrift unter dem Bild an.
+
+Blockbilder werden häufig mit einer Infobox<!-- TODO: link me! --> zusammen formatiert, um die anfängliche Größe zu reduzieren und damit diese sich auf der Seite zu anderem Inhalt befinden:
+
+```markdown
+::: Infobox
+![](img/mod-response.png "Ein Beispiel einer Antwort eines Moderators")
+:::
+```
+
+### Bild-Caching
+
+Bilder werden bis zu 60 Tage im Cache gespeichert. Das Bild wird mit der URL des Bildlinks zugeordnet.
+
+Wenn ein Bild aktualisiert wird, dann muss entweder der Bildname verändert oder ein Suchbegriff an die URL angehängt werden. In beiden Fällen müssen alle Übersetzungen mit dem neuen Bildlink versehen werden.
+
+### Formate und Qualität
+
+Bilder sollten das Format JPG bei einer Qualität von 8 (80 oder 80%, abhängig vom Programm) verwenden. Wenn das Bild (teilweise) transparent ist oder lesbaren Text enthält, dann muss stattdessen das Format PNG benutzt werden. Wenn das Bild eine Animation enthält, kann das GIF eingesetzt werden; jedoch sollte dies nur sparsam verwendet werden, da diese länger zum Laden benötigen und auch größer als die [maximale Dateigröße](#dateigröße) sein können.
```

Comment:
```suggestion
Bilder sollten das Format JPG bei einer Qualität von 8 (80 oder 80%, abhängig vom Programm) verwenden. Wenn das Bild (teilweise) transparent ist oder lesbaren Text enthält, dann muss stattdessen das Format PNG benutzt werden. Wenn das Bild eine Animation enthält, kann das Format GIF eingesetzt werden; jedoch sollte dies nur sparsam verwendet werden, da diese länger zum Laden benötigen und auch größer als die [maximale Dateigröße](#dateigröße) sein können.
```

Reply:
GIF (Graphics Interchange Format) already has `Format` in its name, so I thought that writing something like _das Format Graphics Interchange Format_ is wrong.

Reply:
hmmm actually, i guess technically it would be correct without the `Format`, just sounded better to me xd

Reply:
another solution might be `GI-Format`? Though I'm not sure if everyone will understand this at a glance then because GIF is so widely used.

Reply:
just GIF is good then